api/include/mfxdefs.h

```diff
@@ -50,6 +50,29 @@ extern "C"
 {
 #endif /* __cplusplus */
 
+#if defined(_MSC_VER)
+    #define PRAGMA_IMPL(x) __pragma(x)
+#else
+    #define PRAGMA_IMPL(x) _Pragma(#x)
+#endif
+
+#define PACK_BEGIN_X(x) PRAGMA_IMPL(pack(push, x))
```

Comment:
Shouldn't macro names be prefixed with MFX_ to avoid interleaving with something user-level defined?

Reply:
Well, probably yes

Reply:
done